components/logins/src/login.rs

```diff
@@ -299,6 +322,45 @@ fn string_or_default(row: &Row<'_>, col: &str) -> Result<String> {
 }
 
 impl Login {
```

Comment:
This code feels kind of yucky to me with all the `clone()` calls, but I wasn't sure how to improve it.  Other options I thought about, but didn't go with:

  - Manually implement deserialize (seems like a lot of work)
  - Skip `from_record()` and `into_record()` and get things from the `data` field directly
 
Is there a better way?

Reply:
If I'm understanding right, another possibility is doing a From/Into: https://github.com/mozilla/application-services/blob/main/components/logins/src/login.rs#L549 for your impl and this will actually prevent needing to call `self.guid, etc` (and consequently prevent needing to call clone)

Reply:
We can't use From, since the method also inputs the `EncryptorDecryptor`.  But thanks for the suggestion because it made me realized that Into() consumes the caller, which makes it possible to avoid the cloning.